.github/workflows/cd.yaml

```diff
@@ -27,9 +27,8 @@ jobs:
           show-progress: false
       - name: Install dependencies
         run: |
-          sudo rm /etc/apt/sources.list.d/* && sudo dpkg --clear-avail # Speed up installation and get rid of unwanted lists
           sudo apt-get update
-          sudo apt-get install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev
+          sudo apt-get install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev fuse
```

Comment:
My understanding is that fuse3 is newer & preferred, is fuse required?

Reply:
Do they have separate packages now? That's annoying. Will stuff packaged with fuse 3 work with fuse 2 clients?

Reply:
I'm pretty sure there's some changes to the .so . I know from experience, I have a app that uses fuse2 and it's a hassle

Reply:
```suggestion
          sudo apt-get install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev fuse3
```

Reply:
Couldn't get it to work with fuse 3.